app.js

```diff
@@ -1,23 +1,53 @@
 // Task 1 - Notes for JS Evaluation Predictions
-
+ 
 // List 3 expressions that surprised you or gave unexpected results, why?
+    // 2 + 'pizza'
+    // I didn't anticipate that the number and string would be added into the same string
+
+    // 10 % 2 === 0
```

Comment:
The modulo is looking for the remainder of a division operation. It's actually a 3 step process.

```js
10 / 2 = 5 // <-- the dividend (10) divided by the divisor (5)
5 * 2 = 10 // <-- the result of the previous step (5) times the divisor (2)
10 - 10 = 0 // <-- the result (10) subtracted from the dividend (10). This is the final answer
```